pages/linux/medusa.md

```diff
@@ -0,0 +1,15 @@
+# Medusa
+
+> A speedy, parallel, and modular, login brute-forcer.
```

Comment:
We tend to omit adjectives ("speedy", "top quality") in the main description. Stick to describing what it is and what it does. "Parallel" and "Modular" is ok here in my opinion because it only describes a feature of the tool.
I've left a suggestion below. 
```suggestion
> A modular and parallel login brute-forcer for a variety of protocols.
```